env/WCOSS2.env

```diff
@@ -272,12 +272,12 @@ elif [[ "${step}" = "postsnd" ]]; then
     export OMP_NUM_THREADS=1
 
     export NTHREADS_POSTSND=${NTHREADS1}
-    export mpmd_opt="-ppn 21 ${mpmd_opt}"
-
     export NTHREADS_POSTSNDCFP=${threads_per_task_postsndcfp:-1}
     [[ ${NTHREADS_POSTSNDCFP} -gt ${max_threads_per_task} ]] && export NTHREADS_POSTSNDCFP=${max_threads_per_task}
     export APRUN_POSTSNDCFP="${launcher} -np ${ntasks_postsndcfp} ${mpmd_opt}"
 
+    export mpmd_opt="-ppn 21 ${mpmd_opt}"
```

Comment:
Should this be set to ${tasks_per_node} instead of 21?

Reply:
Fixed it, thanks!